src/modules/navigator/rtl.cpp

```diff
@@ -300,6 +300,11 @@ RTL::set_rtl_item()
 
 	reset_mission_item_reached();
 
+	/* execute command if set. This is required for commands like VTOL transition */
+	if (!item_contains_position(&_mission_item)) {
+		issue_command(&_mission_item);
```

Comment:
I will keep it for now since it's more readable. Let's hope the compiler is smart..